NitroxClient/Communication/Packets/Processors/CoffeeMachineUseProcessor.cs

```diff
@@ -0,0 +1,56 @@
+using NitroxClient.Communication.Packets.Processors.Abstract;
+using NitroxClient.GameLogic;
+using NitroxClient.MonoBehaviours;
+using NitroxModel.GameLogic.FMOD;
+using NitroxModel.Packets;
+using UnityEngine;
+using static NitroxModel.Packets.CoffeeMachineUse;
+
+namespace NitroxClient.Communication.Packets.Processors;
+
+public sealed class CoffeeMachineUseProcessor : ClientPacketProcessor<CoffeeMachineUse>
+{
+    private readonly LocalPlayer localPlayer;
+    private readonly float machineSoundRange;
+
+    public CoffeeMachineUseProcessor(LocalPlayer localPlayer, FMODWhitelist soundWhitelist)
+    {
+        this.localPlayer = localPlayer;
+        soundWhitelist.TryGetSoundData("event:/sub/base/make_coffee", out SoundData coffeeSoundData);
+        machineSoundRange = coffeeSoundData.Radius;
+    }
+
+    public override void Process(CoffeeMachineUse packet)
+    {
+        if (NitroxEntity.TryGetObjectFrom(packet.Id, out GameObject machineGO))
+        {
+            Log.Warn("Failed to get CoffeeVendingMachine gameobject while processing CoffeeMachineUse packet");
+            return;
+        }
+        if (!machineGO.TryGetComponent<CoffeeVendingMachine>(out CoffeeVendingMachine machine))
+        {
+            Log.Warn("Failed to get CoffeeVendingMachine component while processing CoffeeMachineUse packet");
+            return;
+        }
+        bool bPlaySound = Vector3.Distance(machineGO.transform.position, localPlayer.Body.transform.position) < machineSoundRange;
+
+        if (packet.Slot == CoffeeMachineSlot.ONE)
+        {
+            if (bPlaySound)
+            {
+                machine.waterSoundSlot1.Play();
+            }
+            machine.vfxController.Play(0);
+            machine.timeLastUseSlot1 = Time.time;
+        }
+        else
```

Comment:
Do we want to have an if else here with the second slot and an else logging and error incase they ever add another slot?

Not really needed as I suspect they won't update much more but just a thought